python_modules/libraries/dagster-airlift/dagster_airlift/in_airflow/dagster_run_utils.py

```diff
@@ -0,0 +1,37 @@
+from typing import Any, Mapping, NamedTuple, Optional
+
+TERMINAL_STATI = ["SUCCESS", "FAILURE", "CANCELED"]
+SYSTEM_TAG_PREFIX = "dagster/"
+MAX_RETRIES_TAG = f"{SYSTEM_TAG_PREFIX}max_retries"
+WILL_RETRY_TAG = f"{SYSTEM_TAG_PREFIX}will_retry"
+AUTO_RETRY_RUN_ID_TAG = f"{SYSTEM_TAG_PREFIX}auto_retry_run_id"
+RETRY_NUMBER_TAG = f"{SYSTEM_TAG_PREFIX}retry_number"
+PARENT_RUN_ID_TAG = f"{SYSTEM_TAG_PREFIX}parent_run_id"
+SUCCESS_STATUS = "SUCCESS"
+RETRY_ON_ASSET_OR_OP_FAILURE_TAG = f"{SYSTEM_TAG_PREFIX}retry_on_asset_or_op_failure"
+RUN_FAILURE_REASON_TAG = f"{SYSTEM_TAG_PREFIX}failure_reason"
+STEP_FAILURE_REASON = "STEP_FAILURE"
```

Comment:
any particular reason to not import these from dagster? avoiding private imports?

Reply:
the in-airflow package has no dagster dependency. I'll add a comment